src/applications/claims-status/components/ClaimsContentionList.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { setFocus } from '../utils/page';
+
+const MAX_CONTENTIONS = 3;
+
+function ClaimsContentionList({ contentions, contentionsHeader }) {
+  const ellipsis = useRef(null);
+  const additionalContentions = useRef(null);
+  const viewMoreBtn = useRef(null);
+  const hasContentions = contentions && contentions.length;
+  const contentionsText = hasContentions
+    ? contentions
+        .slice(0, MAX_CONTENTIONS)
+        .map(cond => cond.name)
+        .join(', ')
+    : 'Not available';
+  const additionalContentionsText = hasContentions
+    ? contentions
+        .slice(MAX_CONTENTIONS, contentions.length)
+        .map(cond => cond.name)
+        .join(', ')
+    : false;
```

Comment:
Instead of generating 2 lists here, I think we would be better of creating a helper function that we can pass the contentions to and the number of contentions we want to render instead. Something like:
```js
const renderContentions = (contentions, limit) => {
     // ...
};

Then we can dynamically determine the `limit` based on the `useState`-managed flag mentioned in one of the above comments